defconfig/ADKU3.SDRAM.XSIM.hls_hashjoin.defconfig

```diff
@@ -27,7 +27,7 @@ HLS_SUPPORT="TRUE"
 # ENABLE_BRAM is not set
 BRAM_USED="FALSE"
 ENABLE_SDRAM=y
-SDRAM_USED="TRUE"
+SDRAM_USED="FALSE"
```

Comment:
If you want to disable SDRAM you need to create a new .defconfig file. You need to remove the term SDRAM from the filename and the line "ENABLE_SDRAM=y" from the file content.